scripts/generate-PR-body.sh

```diff
@@ -0,0 +1,24 @@
+echo "pr_title=Sync with ethereum/solidity@develop $(date +%d-%m-%Y)" >> $GITHUB_ENV
```

Comment:
not sure why but it works: https://github.com/solidity-docs-test/pl-polish/pull/19

Reply:
ok, maybe there's some magic done by github.

Reply:
after your suggestion, PR title is quoted: https://github.com/solidity-docs-test/pl-polish/pull/34. I think better was without it.

Reply:
Yeah, it should not be quoted. Github is apparently doing some custom parsing here rather than just passing those instructions through `eval` as I expected.

This is really why I prefer to keep any CI-specific syntax in the .yaml file. It looks like shell code but it's just github's own odd thing.